packages/core/src/payment/strategies/credit-card-redirect/credit-card-redirect-payment-strategy.ts

```diff
@@ -65,14 +65,14 @@ export default class CreditCardRedirectPaymentStrategy extends CreditCardPayment
                 return Promise.reject(error);
             }
 
-            return new Promise(() =>
-                this._formPoster.postForm(error.body.three_ds_result.acs_url, {
-                    PaReq: error.body.three_ds_result.payer_auth_request || null,
-                    TermUrl: error.body.three_ds_result.callback_url || null,
-                    MD: error.body.three_ds_result.merchant_data || null,
-                }),
-            );
+            this._formPoster.postForm(error.body.three_ds_result.acs_url, {
+                PaReq: error.body.three_ds_result.payer_auth_request || null,
+                TermUrl: error.body.three_ds_result.callback_url || null,
+                MD: error.body.three_ds_result.merchant_data || null,
+            });
         }
+
+        return this._store.dispatch(this._orderActionCreator.loadCurrentOrder());
```

Comment:
Add this line in order to return `Promise<InternalCheckoutSelectors>`.

Reply:
Does loadCurrentOrder make API call?

Reply:
Yes, it will.
We have a same line on L112 of this file.